recipe/meta.yaml

```diff
@@ -35,10 +35,11 @@ test:
 
 about:
   home: https://github.com/tqdm/tqdm
-  license: Mozilla Public License 2.0 (MPL 2.0) or MIT License
+  license: MPLv2.0, MIT
   summary: 'A Fast, Extensible Progress Meter'
 
 extra:
   recipe-maintainers:
+    - casperdcl
```

Comment:
> but we're happy with individual access, thanks.

It does not mean we are against it. We just do not have the manpower to check the what needs to be done in the bot scripts to handle that. (Maybe it is a low hanging fruit.)